addons/mail/views/res_company_views.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo>
+    <data>
```

Comment:
Note that these views have a data tag but not the ones in "mail_alias_domain_views" 👀